urllib3/response.py

```diff
@@ -126,8 +129,8 @@ def __init__(self, body='', headers=None, status=0, version=0, reason=None,
         # Are we using the chunked-style of transfer encoding?
         self.chunked = False
         self.chunk_left = None
-        tr_enc = self.headers.get('transfer-encoding', '')
-        if tr_enc.lower() == "chunked":
+        tr_enc = self.headers.get('transfer-encoding', '').lower()
+        if "chunked" in set(tr_enc.split(", ")):
```

Comment:
Sets will make lookup faster in a case where a server sends a bunch of encodings separated by commas. If you don't want to hedge against that case (which I've heard of happening with TomCat servers from a friend), we can remove it.